test/utils/utils.go

```diff
@@ -61,33 +62,33 @@ func Run(cmd *exec.Cmd) (string, error) {
 }
 
 // InstallPrometheusOperator installs the prometheus Operator to be used to export the enabled metrics.
-func InstallPrometheusOperator() error {
+func InstallPrometheusOperator(ctx context.Context) error {
 	url := fmt.Sprintf(prometheusOperatorURL, prometheusOperatorVersion)
```

Comment:
We are still not making use of this, thanks for updating anyway. We will introduce e2e tests at some point